onnxscript/rewriter/pattern_test.py

```diff
@@ -688,6 +688,39 @@ def test_model(x: FLOAT[1024], y: FLOAT[1024]) -> FLOAT[1024]:
         self.assertEqual(len(model.graph), 2)
         self.assertEqual([x.op_type for x in model.graph], ["Constant", "Identity"])
 
+    def test_or_pattern(self):
+        def source_pattern(op, x, y, bias):
+            t1 = op.MatMul(x, y)
+            t2 = op.Add(t1, bias)
+            t1_or_t2 = pattern.OrValue([t1, t2], tag_var="has_bias", tag_values=[False, True])
+            return op.Relu(t1_or_t2)
+
+        def replacement(op, x, y, bias, has_bias):
+            if has_bias:
+                return op.WithBias(x, y, bias)
+            else:
+                return op.WithoutBias(x, y)
+
+        rule = pattern.RewriteRule(source_pattern, replacement)
+
+        @script()
+        def test_model1(x: FLOAT[16, 32], y: FLOAT[32, 16]) -> FLOAT[16, 16]:
+            return op.Relu(op.MatMul(x, y))
+
+        model_proto = test_model1.to_model_proto()
+        model = ir.serde.deserialize_model(model_proto)
```

Comment:
We can create a util function in this test class to avoid having to call serde everytime.

Reply:
Yes, onnx functions should have a method to return the "ir model" ... in fact, the current version is logically a double-conversion (since script-function => IR => proto should be the logical route, eventually, to create proto).